PyInstaller/exceptions.py

```diff
@@ -11,4 +11,11 @@
 
 
 class ExecCommandFailed(SystemExit):
```

Comment:
I think we actually want a whole new exception type - not just an extension of a pre-existsing one.

Reply:
I see - thanks for letting me know. This class was in the exceptions.py file when I forked it, and was empty - so I assumed that this was the class for overall exceptions, and that individual exceptions should be added as class methods.

Reply:
@RS357 Actually you are hijacking the `ExecCommandFailed` exception, which is already used and has a quite specific meaning - as its name states.

In Python, exceptions are not added as attributes, methods, or such. Exceptions are classes of its own, eventually put into a useful hierarchy using sub-classing.